rules-tests/Visibility/Rector/ClassMethod/ExplicitPublicClassMethodRector/Fixture/fixture.php.inc

```diff
@@ -1,5 +1,32 @@
 <?php
 
+abstract class SomeAbstractClass
+{
+    abstract function noExplicitVis();
+
+    abstract public function publicVis();
+
+    abstract protected function protectedVis();
+
+    abstract private function privateVis();
+
+    static function noExplicitVisStatic()
+    {
+    }
+
+    public static function publicVisStatic()
+    {
+    }
+    
+    protected static function protectedVisStatic()
+    {
+    }
+    
+    private static function privateVisStatic()
+    {
+    }
+}
+
```

Comment:
Could you extract this fixture to own file? That way we have mininal reproducer and can easily backtrack the issue to specific fixture file in case of regression in the future.